lessons/Early-Math/commutative-property/langs/en-us/index.adoc

```diff
@@ -193,7 +202,7 @@ When everyone is finished, check in with students. Did everyone discover that _n
 @slidebreak
 
 
-In the last activity, we discovered that _none_ of the functions on the page were commutative! Do you think there are any functions in @proglang that are commutative? Let's answer that question.
+In the last activity, we discovered that _none_ of the image-producing functions on the page were commutative! Do you think there are any functions in @proglang that are commutative? Let's answer that question.
```

Comment:
You're asking them to answer a simple yes-or-no question, when thinking about the actual answer would be nearly impossible without having more examples of image-producing functions. That means students will just flip a coin and pick "yes" or "no", neither of which is helping them think through stuff.

I'm sorry to disappoint you, but now I'm even more convinced that this section is backwards. Here's how I would set it up:

- Give kids a workbook page with `above`, `beside`, and `overlay`. For each one, have them read the contract, convert a given COE to code, then draw the COE with arguments swapped. Students have to make a prediction about whether they will get the same value back by _drawing the image they predict they'll get back_ when they convert the swapped COE to code. Have them try it out, draw the answer, then circle "commutative" or "not commutative".  At the bottom, leave an open response that asks them if they can imagine an image function that IS commutative
- Page 2: Same formula, but with`string-contains`, `min`, `max`, and `blend-images`